src/lib/api/ipfs.ts

```diff
@@ -81,24 +81,25 @@ export const editMetadataForCid = async (
 }
 
 // TODO after the move to Infura for IPFS, we can probably look at removing this.
-export const ipfsGetWithFallback = async (
+export const ipfsGetWithFallback = async <T>(
   hash: string,
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   { fallbackHostname }: { fallbackHostname?: string } = {},
 ) => {
-  // try {
   // Build config for axios get request
-  const response = await axios({
-    method: 'get',
-    url: restrictedIpfsUrl(hash),
+  const response = await axios.get<T>(restrictedIpfsUrl(hash), {
     responseType: 'json',
     headers: {
       Accept: 'application/json',
       'Content-Type': 'application/json',
     },
   })
-  if (response.data.Data?.['/'].bytes) {
-    response.data = extractJsonFromBase64Data(response.data.Data['/'].bytes)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  if ((response.data as any).Data?.['/'].bytes) {
```

Comment:
**question** is `response.data` of type `T` now? Is there a way to avoid `any`?

Reply:
seems like this is something we can remove (cc @wraeth-eth) so i've left it in awkward state. but yes let's plz avoid `any`

Reply:
This was to get around all the ipfs chicanery intro'd by a bug in pinata ipfs upgrade. I think we should open an issue to investigate it (Don't mind doing it) and leave here for now

Reply:
https://github.com/jbx-protocol/juice-interface/issues/2886

Reply:
Just had a look; this is still a problem. We'll have to wait for pinata to fix it (if they ever do)